### PR TITLE
Add warning info to install prereqs

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
 
 If you are already using a PaaS solution, MuleSoft recommends deploying Runtime Fabric in parallel with your PaaS. This enables you to take advantage of the complete benefits of Anypoint Platform.
 
+== Shared Responsibility for On-Premises Runtime Fabric
+
+The successful operation of on-premises Runtime Fabric instances is a shared responsibility. It is critical to understand which areas you must manage and which areas are managed by MuleSoft. Without the required infrastructure and proper maintenance, Runtime Fabric will not operate successfully.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:30]: Nice intro!

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
+
+The following image illustrates MuleSoft and customer responsibilities for on-premises Runtime Fabric instances:
+
+image::sharedresponsibility.png[Runtime Fabric Shared Responsiblities]
+
+* MuleSoft responsibility - MuleSoft manages the RTF control plane and is responsible for the delivered components, Runtime Fabric appliance, Runtime Fabric agent, Mule runtime engine, and other dependencies for Mule appications.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]: Format per style ref.; also, never use the abbreviation RTF, also per style ref. and PMM Product Tiering & Hierarchy:
```suggestion
* MuleSoft responsibility +
MuleSoft manages the Runtime Fabric control plane and is responsible for the delivered components, Runtime Fabric appliance, Runtime Fabric agent, Mule runtime engine, and other dependencies for Mule applications.
```

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
+
+* Customer responsibility - Customers are responsible for provisioning, configuring, and managing the infrastructure required for Runtime Fabric and Runtime Fabric clusters.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* Customer responsibility +
Customers are responsible for provisioning, configuring, and managing the infrastructure required for Runtime Fabric and Runtime Fabric clusters, including:  
```

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
+
+The following image illustrates MuleSoft and customer responsibilities for on-premises Runtime Fabric instances:
+
+image::sharedresponsibility.png[Runtime Fabric Shared Responsiblities]
+
+* MuleSoft responsibility - MuleSoft manages the RTF control plane and is responsible for the delivered components, Runtime Fabric appliance, Runtime Fabric agent, Mule runtime engine, and other dependencies for Mule appications.
+
+* Customer responsibility - Customers are responsible for provisioning, configuring, and managing the infrastructure required for Runtime Fabric and Runtime Fabric clusters.  
++
+** Infrastructure includes:
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:44]: Omit and promote subsequent text units:
```suggestion
```

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
++
+*** VM resources (CPU, Memory)
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:46]:
```suggestion
* VM resources (CPU, memory)
```

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
++
+*** Disk performance and capacity
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:48]:
```suggestion
* Disk performance and capacity
```

File: modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
++
+*** Operating systems and kernel patching
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:50]:
```suggestion
* Operating systems and kernel patching
```

File: modules/ROOT/pages/index.adoc [hunk 8]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
++
+*** Network ports
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:52]:
```suggestion
* Network ports
```

File: modules/ROOT/pages/index.adoc [hunk 9]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
++
+*** Syncing system time across all VMs
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]:
```suggestion for parallelism
* Synchronization of system time across all VMs
```
OR
```suggestion
* Synchronized system time across all VMs
```

File: modules/ROOT/pages/index.adoc [hunk 10]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
++
+** For infrastructure provisioning and management, you will need the assistance of your organization’s:
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]: Per the edits above, this follows:
```suggestion
For infrastructure provisioning and management, you need the assistance of the following teams in your organization:
```

File: modules/ROOT/pages/index.adoc [hunk 11]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
++
+*** DevOps team to provision and manage the infrastructure
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:58]:
```suggestion
* DevOps, to provision and manage the infrastructure
```

File: modules/ROOT/pages/index.adoc [hunk 12]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
++
+*** Network team to whitelist ports and configure proxy settings
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]: Per inclusive language initiative (see style ref):
```suggestion
* Network, to allowlist ports and configure proxy settings
```
OR
```suggestion
* Network, to specify allowed ports and configure proxy settings
```

File: modules/ROOT/pages/index.adoc [hunk 13]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
++
+*** Security team to review for compliance and obtain security certificates
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* Security, to review compliance and obtain security certificates
```

File: modules/ROOT/pages/index.adoc [hunk 14]
```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
+
+Refer to xref:install-prereqs.adoc[Runtime Fabric Installation Prerequisites] for additional information.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:64]: Per our new linking policies, only need-to-know-right-this-minute xrefs appear in main narrative and all others go into a ==See Also subtopic at the end of the topic set.
Also per policy, if we provide a reason for the xref, it should be more robust than "for additional information": for information about what, specifically, and to what user purpose?

File: modules/ROOT/pages/install-aws.adoc [hunk 15]
```diff
@@ -7,10 +7,10 @@ This topic describes how to install Anypoint Runtime Fabric on your AWS account.
 
 [WARNING]
 ====
-Before installing Runtime Fabric, ensure your infrastructure meets the minimum hardware, operating system, 
-and networking requirements.
+Before installing Runtime Fabric, refer to xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric]. If your infrastructure does not meet the minimum hardware, operating system,
```
Comment (kbostrom) [inline modules/ROOT/pages/install-aws.adoc:10]: Front-load the xref:
```suggestion
Before installing Runtime Fabric, ensure that your infrastructure meets all of the minimum hardware, operating system, and networking requirements by referring to refer to xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric]; otherwise, Runtime Fabric will not operate successfully.
```
OR
```suggestion
Before installing Runtime Fabric, ensure that your infrastructure meets all of the minimum hardware, operating system, and networking requirements (xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric]), or Runtime Fabric will not operate successfully.
```

File: modules/ROOT/pages/install-aws.adoc [hunk 16]
```diff
@@ -7,10 +7,10 @@ This topic describes how to install Anypoint Runtime Fabric on your AWS account.
 
 [WARNING]
 ====
-Before installing Runtime Fabric, ensure your infrastructure meets the minimum hardware, operating system, 
-and networking requirements.
+Before installing Runtime Fabric, refer to xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric]. If your infrastructure does not meet the minimum hardware, operating system, 
+and networking requirements, Runtime Fabric will not operate successfully.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-aws.adoc:11]: included in above ^
```suggestion
```

File: modules/ROOT/pages/install-aws.adoc [hunk 17]
```diff
@@ -7,10 +7,10 @@ This topic describes how to install Anypoint Runtime Fabric on your AWS account.
...
 
-Your organization's operations, networking and security teams should review these requirements. See 
+Your organization's operations, networking and security teams must be involved. See
```
Comment (kbostrom) [inline modules/ROOT/pages/install-aws.adoc]:
```suggestion
Your organization's operations, networking, and security teams must be involved. See 
```
But should we use the exact same names as used earlier: DevOps, Network, and Security?

File: modules/ROOT/pages/install-azure.adoc [hunk 18]
```diff
@@ -8,11 +8,11 @@ This topic describes how to install Anypoint Runtime Fabric on your Azure accoun
 
 [WARNING]
 ====
-Before installing Runtime Fabric ensure your infrastructure meets the minimum hardware, operating system, 
-and networking requirements.
+Before installing Runtime Fabric, refer to xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric]. If your infrastructure does not meet the minimum hardware, operating system,
```
Comment (kbostrom) [inline modules/ROOT/pages/install-azure.adoc:11]: ditto prev. edit to same text earlier

File: modules/ROOT/pages/install-azure.adoc [hunk 19]
```diff
@@ -8,11 +8,11 @@ This topic describes how to install Anypoint Runtime Fabric on your Azure accoun
...
 [WARNING]
 ====
-Before installing Runtime Fabric ensure your infrastructure meets the minimum hardware, operating system, 
-and networking requirements.
+Before installing Runtime Fabric, refer to xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric]. If your infrastructure does not meet the minimum hardware, operating system, 
+and networking requirements, Runtime Fabric will not operate successfully.
 
-Your organization's operations, networking and security teams should review these requirements. 
-See xref:install-prereqs.adoc[Anypoint Runtime Fabric Installation Prerequisites].
+Your organization's operations, networking and security teams must be involved. See
```
Comment (kbostrom) [inline modules/ROOT/pages/install-azure.adoc]: ditto prev

File: modules/ROOT/pages/install-manual.adoc [hunk 20]
```diff
@@ -7,7 +7,11 @@ This topic describes how to install and register Anypoint Runtime Fabric on VMs
 
 [WARNING]
 ====
-Ensure that your infrastructure meets the minimum hardware, operating system, and networking requirements. Your organization's operations, networking, and security teams should review these requirements. See xref:install-prereqs.adoc[Anypoint Runtime Fabric Installation Prerequisites].
+Before installing Runtime Fabric, refer to xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric]. If your infrastructure does not meet the minimum hardware, operating system, 
+and networking requirements, Runtime Fabric will not operate successfully.
+
+Your organization's operations, networking and security teams must be involved. See 
+xref:install-prereqs.adoc[Anypoint Runtime Fabric Installation Prerequisites].
```
Comment (kbostrom) [inline modules/ROOT/pages/install-manual.adoc]: ditto prev

File: modules/ROOT/pages/install-prereqs.adoc [hunk 21]
```diff
@@ -4,12 +4,32 @@ include::_attributes.adoc[]
 endif::[]
 :page-aliases: install-port-reqs.adoc, install-sys-reqs.adoc
 
-Review the following requirements with your operations and IT teams to ensure that all prerequisites are met before installing Anypoint Runtime Fabric. The Runtime Fabric installation process fails if the minimum requirements are not met.
+== Before You Begin
+
+[IMPORTANT]
```
Comment (kbostrom) [inline modules/ROOT/pages/install-prereqs.adoc:9]: I don't think we need this notice title here, because prereqs are prereqs and by nature important.
Also, such a notice at the very beginning of the topic, comprising the topic, is not standard.

Comment (kbostrom) [reply modules/ROOT/pages/install-prereqs.adoc:9]: Please consider previous edits to this text and whether you want to word this exactly the same way.
I understand that this text is actually the first that the audience sees about these topics, but you might not need to expand the narrative.
Copyedits only below, based on consideration of previous wording edits...:

File: modules/ROOT/pages/install-prereqs.adoc [hunk 22]
```diff
@@ -4,12 +4,32 @@ include::_attributes.adoc[]
...
+Most Anypoint Runtime Fabric issues result from improper infrastructure provisioning or configuration. 
+
+Before you install Runtime Fabric :
```
Comment (kbostrom) [inline modules/ROOT/pages/install-prereqs.adoc:12]: c/b just one para
Also, remove space"
```suggestion
Most Anypoint Runtime Fabric issues result from improper infrastructure provisioning or configuration. Before you install Runtime Fabric:
```

File: modules/ROOT/pages/install-prereqs.adoc [hunk 23]
```diff
@@ -4,12 +4,32 @@ include::_attributes.adoc[]
...
+If your infrastructure does not meet the minimum hardware, operating system, and networking requirements, Runtime Fabric will not operate successfully.
+
+* Enlist the assistance of your organization’s:
+** DevOps team to provision and manage the infrastructure
+** Network team to whitelist ports and configure proxy settings
+** Security team to review for compliance and obtain security certificates
++
+Working with these teams in your organization helps ensure a successful installation and avoids performance and density issues. 
+
+* Review the following requirements and ensure that all prerequisites are met before installing Anypoint Runtime Fabric.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-prereqs.adoc:25]: Which "following requirements"?
Do you mean subsequent subtopics?
If so, avoid using language in one topic that refers to another, in case topics are reused or the audience is not reading everything chronologically.
I suggest something more like...:
```suggestion
* Ensure that you have the general knowledge required to perform and installation [and whatever topics follow that comprise "the following requirements"].
```

**Review by kbostrom — DISMISSED** [review]
Edited

**Review by dhirsch1 — APPROVED** [review]

**Comment by sundippatel** [comment]
Adding a more clear diagram:
https://docs.google.com/presentation/d/1yHaO0Sg5X0YwjbqsdXvaA13JYZ9RVTWK6sI6ZmXAzPI/edit#slide=id.g8ec7c922d3_0_0

File: modules/ROOT/pages/index.adoc [hunk 24]
```diff
@@ -25,6 +25,44 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
 
 If you are already using a PaaS solution, MuleSoft recommends deploying Runtime Fabric in parallel with your PaaS. This enables you to take advantage of the complete benefits of Anypoint Platform.
 
+== Shared Responsibility for On-Premises Runtime Fabric
+
+The successful operation of on-premises Runtime Fabric instances is a shared responsibility. It is critical to understand which areas you must manage and which areas are managed by MuleSoft. Without the required infrastructure and proper maintenance, Runtime Fabric will not operate successfully.
```
Comment (sundippatel) [inline modules/ROOT/pages/index.adoc:30]: Suggest rewording the last sentence to be more positive in light of RTF.

"Runtime Fabric will only operate successfully if it's provided the required infrastructure and maintenance."

File: modules/ROOT/pages/index.adoc [hunk 25]
```diff
@@ -25,6 +25,44 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
+
+The following image illustrates MuleSoft and customer responsibilities for on-premises Runtime Fabric instances:
+
+image::sharedresponsibility.png[Runtime Fabric Shared Responsiblities]
+
+* MuleSoft responsibility:
++
+MuleSoft manages the Runtime Fabric control plane and is responsible for the delivered components, Runtime Fabric appliance, Runtime Fabric agent, Mule runtime engine, and other dependencies for Mule appications.
```
Comment (sundippatel) [inline modules/ROOT/pages/index.adoc:38]: "Runtime Fabric control plane" isn't a term we use. Replace with Runtime Fabric appliance, perhaps?

File: modules/ROOT/pages/index.adoc [hunk 26]
```diff
@@ -25,6 +25,44 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
...
+
+* Customer responsibility:
++
+Customers are responsible for provisioning, configuring, and managing the infrastructure required for Runtime Fabric and Runtime Fabric clusters.
```
Comment (sundippatel) [inline modules/ROOT/pages/index.adoc:42]: Runtime Fabric and Runtime Fabric clusters is redundant. Suggest removing "Runtime Fabric clusters"

Comment (paigehargrave) [reply modules/ROOT/pages/index.adoc:42]: Updated this - thank you!

File: modules/ROOT/pages/index.adoc (hunk 25 above)
Comment (paigehargrave) [reply modules/ROOT/pages/index.adoc:38]: Updated this - thank you!

File: modules/ROOT/pages/index.adoc (hunk 24 above)
Comment (paigehargrave) [reply modules/ROOT/pages/index.adoc:30]: Updated this - thank you!
